ion/bitstream_test.go

```diff
@@ -109,3 +112,188 @@ func TestBitcodeString(t *testing.T) {
 		}
 	}
 }
+
+func TestBinaryReadTimestamp(t *testing.T) {
+	test := func(ion []byte, eval string) {
```

Comment:
I think `eval` is intended to mean `expectedValue`, but the word `eval` is widely used to mean `evaluate` or `evaluation`. Could you expand it?

Reply:
I used an `e` prefix to mean "expected" throughout the test code, including a lot of `eval`s for "expected value", so this one is presumably my fault. :) I prefer the brevity, especially for locally-scoped variables where I don't have to go far to see how they get used, but you're not wrong and if you want to expand them go for it.

Reply:
Updated to `expectedValue`